react/week3/react-w3/src/List.js

```diff
@@ -0,0 +1,26 @@
+import Item from "./Item.js";
+
+function List({ todos, setTodos }) {
+  function remove(id) {
+    setTodos(todos.filter((item) => item.id !== id));
+  }
```

Comment:
Same comment as in the AddTodo. I would have the remove function in App.js, and passed it down here, instead of passing down the setTodos